gradle.properties

```diff
@@ -1,12 +1,12 @@
 org.gradle.caching=true
 group=io.ballerina.stdlib
 version=1.6.2-SNAPSHOT
-ballerinaLangVersion=2201.8.0
+ballerinaLangVersion=2201.10.0-SNAPSHOT
```

Comment:
Shall we generate a new lang timestamp with java21 and update it here? It will fix the build failure

Reply:
triggered a lang build https://github.com/ballerina-platform/ballerina-lang/actions/runs/11047985959

Reply:
Should we done that finally. PR Build will anyway use lang version which was build

Reply:
I think the PR build pick from the `gradle.properties`. So it might fail for `2201.10.0-SNAPSHOT`

Reply:
Updated with timestamp.